### PR TITLE
Remove RSpec

### LINKED ISSUE #151: Complete migration to Minitest
This gem currently uses both minitest and rspec to test. I'd like to combine all tests to use only one or the other. @wvanbergen I can see that you started moving all tests to minitest a long time ago. Is this still what you want?

### PR DESCRIPTION
**Pull request by mvz**
Unify all testing under `test/` using `minitest` and `minitest/spec`.

Fixes #151.

### CODE REVIEW

File: adyen.gemspec [hunk 1]
```diff
@@ -39,5 +38,5 @@ Gem::Specification.new do |s|
   s.extra_rdoc_files = ['README.md', 'CHANGELOG.md', 'CONTRIBUTING.md']
 
   s.files = `git ls-files`.split($/)
-  s.test_files = s.files.grep(%r{^(test|spec|features)/})
```
Comment (wvanbergen) [inline adyen.gemspec]: This is the standard way bundler generates the gemspec file which I just left as it was generated. If you're going to remove `spec`, you can remove `features` as well

File: lib/adyen/api/payment_service.rb [hunk 2]
```diff
@@ -258,6 +258,7 @@ class AuthorisationResponse < Response
         response_attrs :result_code, :auth_code, :refusal_reason, :psp_reference,
           :additional_data
 
+        # TODO: Avoid overriding this method
```
Comment (wvanbergen) [inline lib/adyen/api/payment_service.rb]: ?

File: test/functional/api_test.rb [hunk 3]
```diff
@@ -1,6 +1,7 @@
 # encoding: UTF-8
-require 'api/spec_helper'
+require 'test_helper'
```
Comment (wvanbergen) [inline test/functional/api_test.rb]: Would `require 'unit/api/test_helper'` work here?

File: adyen.gemspec (hunk 1 above)
Comment (mvz) [inline adyen.gemspec]: Updated.

**Comment by mvz** [comment]
The failures are the same as on current master, and seem to have to do with permissions on the Adyen test account Travis CI uses.

**Pull request merged**